packages/okta-adapter/src/definitions/fetch/fetch.ts

```diff
@@ -116,6 +116,30 @@ const getPrivateAPISettingsDefinitions = ({
   }
 }
 
+const brandCustomizer: definitions.fetch.FetchTopLevelElementDefinition['elemID'] = {
+  custom:
+    args =>
+    ({ entry, defaultName }) => {
+      if (entry?.isDefault) {
```

Comment:
We prefer explicit checks
```suggestion
      if (entry?.isDefault === true) {
```

Reply:
Done